docs/drafts/upgrade-support.md

```diff
@@ -1,33 +1,58 @@
 
-* [Semantic Versioning](https://semver.org/) (Semver)
-* [Legacy OLM 0 semantics](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/#methods-for-specifying-updates): the `replaces`/`skips`/`skipRange` directives
+By supporting legacy OLM semantics, OLM v1 now honors the upgrade graph from catalogs accurately.
 
-The Kubernetes manifests in this repo enable legacy support by default. Cluster admins can control which semantics to use by passing one of the following arguments to the `manager` binary:
-* `--feature-gates=ForceSemverUpgradeConstraints=true` - enable Semver
-* `--feature-gates=ForceSemverUpgradeConstraints=false` - disable Semver, use legacy semantics
+* If there are multiple possible successors, OLM v1 behavior differs in the following ways:
+  * In legacy OLM, the successor closest to the channel head is chosen.
+  * In OLM v1, the successor with the highest semantic version (semver) is chosen.
+* Consider the following set of file-based catalog (FBC) channel entries:
 
-For example, to enable Semver update the `controller-manager` Deployment manifest to include the following argument:
+  ```yaml
+  # ...
+  - name: example.v3.0.0
+    skips: ["example.v2.0.0"]
+  - name: example.v2.0.0
+    skipRange: >=1.0.0 <2.0.0
+  ```
 
-```yaml
-- command:
-  - /manager
-  args:
-  - --feature-gates=ForceSemverUpgradeConstraints=true
-  image: controller:latest
+If `1.0.0` is installed, OLM v1 behavior differs in the following ways:
+
+  * Legacy OLM does not detect an upgrade edge to `v2.0.0` because `v2.0.0` is skipped and not on the `replaces` chain.
+  * OLM v1 detects the upgrade edge because OLM v1 does not have a concept of a `replaces` chain. OLM v1 finds all entries that have a `replace`, `skip`, or `skipRange` value that covers the currently installed version.
+
+You can change the default behavior of the upgrade constraints by setting the `upgradeConstraintPolicy` parameter in your cluster extension's custom resource (CR).
+
+``` yaml hl_lines="10"
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterExtension
+metadata:
+  name: <extension_name>
+spec:
+  installNamespace: <namespace>
+  packageName: <package_name>
+  serviceAccount:
+    name: <service_account>
+  upgradeConstraintPolicy: Ignore
+  version: "<version_or_version_range>"
 ```
 
-In a future release, it is planned to remove the `ForceSemverUpgradeConstraints` feature gate and allow package authors to specify upgrade constraint semantics at the catalog level.
+where setting the `upgradeConstraintPolicy` to:
+
+`Ignore`
+:   Does not limit the next version to the set of successors, and instead allows for any downgrade, sidegrade, or upgrade.
+
+`Enforce`
```

Comment:
Another upcoming change: https://github.com/operator-framework/operator-controller/issues/1151

Reply:
@joelanford Is this something we would want to change to the new value now? Or do we want to update #1151 with a comment or a docs issue?

Reply:
Use what you have, which is what is on `main`. The PR for that change should hopefully find/replace these.